src/elfpy/pricing_models/yieldspace.py

```diff
@@ -545,7 +549,7 @@ def calc_out_given_in(
             and the unit of the tokens).
         market_state : MarketState
             The state of the AMM's reserves and share prices.
-        fee_percent : float
+        trade_fee_percent : float
             The percentage of the difference between the amount paid without
             slippage and the amount received that will be added to the input
             as a fee.
```

Comment:
Again add `redemption_fee_percent` to docstring pls

Reply:
yup!